srcpkgs/samba/template

```diff
@@ -1,137 +1,190 @@
 # Template file for 'samba'
 pkgname=samba
-version=3.6.25
-revision=15
-build_wrksrc=source3
-build_style=gnu-configure
-configure_args="--with-fhs --with-pam --with-pam_smbpass --with-ldap
- --with-configdir=/etc/samba --with-lockdir=/var/samba --with-static-libs=libtevent
- --with-cachedir=/var/samba --with-nmbdsocketdir=/run/samba/socket
- --with-statedir=/var/samba --with-piddir=/run/samba --with-dnsupdate
- --with-pammodulesdir=/usr/lib/security --with-automount --disable-swat
- --with-syslog --enable-external-libtalloc --with-quotas --enable-cups
- --enable-external-libtdb --disable-fam --with-ads --with-acl-support
- --with-shared-modules=idmap_ad,idmap_adex,idmap_rid,idmap_hash,idmap_tdb2
- --sbindir=/usr/bin"
+version=4.13.2
+revision=1
+build_style=waf3
+build_helper="qemu"
+configure_script="buildtools/bin/waf"
+_idmap_modules="idmap_ad,idmap_rid,idmap_adex,idmap_hash,idmap_tdb2"
+_pdb_modules="pdb_tdbsam,pdb_ldap,pdb_ads,pdb_smbpasswd,pdb_wbc_sam,pdb_samba4"
+_auth_modules="auth_unix,auth_wbc,auth_server,auth_netlogind,autH_script,auth_samba4"
+configure_args="--enable-fhs --sbindir=/usr/bin --localstatedir=/var
+ --sysconfdir=/etc --with-piddir=/run/samba --with-sockets-dir=/run/samba
+ --with-privatedir=/etc/samba/private --with-pammodulesdir=/usr/lib/security
+ --with-modulesdir=/usr/lib/samba --disable-rpath --disable-rpath-install
+ --without-systemd --without-gettext --bundled-libraries=NONE
+ --with-system-mitkrb5 --without-ad-dc
+ --with-shared-modules=${_idmap_modules},${_pdb_modules},${_auth_modules}"
+hostmakedepends="pkg-config perl-Parse-Yapp rpcsvc-proto docbook2x
+ libtasn1-tools tdb-python3 tevent-python3 talloc-python3"
+makedepends="python3-devel libtirpc-devel popt-devel e2fsprogs-devel
+ mit-krb5-devel pam-devel acl-devel cups-devel avahi-libs-devel tdb-devel
+ talloc-devel tevent-devel ldb-devel cmocka-devel gnutls-devel zlib-devel
+ ncurses-devel libldap-devel libarchive-devel jansson-devel"
 short_desc="SMB/CIFS file, print, and login server for Unix"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-3"
+license="GPL-3.0-or-later"
 homepage="http://www.samba.org"
-distfiles="http://us1.samba.org/samba/ftp/stable/$pkgname-$version.tar.gz"
-checksum=8f2c8a7f2bd89b0dfd228ed917815852f7c625b2bc0936304ac3ed63aaf83751
-
+distfiles="http://download.samba.org/pub/samba/stable/${pkgname}-${version}.tar.gz"
+checksum=276464396a05d88b775bda01ac2eb1e5a636ccf7010b0fd28efc3d85583af2b4
 lib32disabled=yes
 conf_files="/etc/pam.d/samba /etc/samba/smb.conf"
 make_dirs="/etc/samba/private 0750 root root"
-hostmakedepends="pkg-config perl python-devel"
-makedepends="readline-devel libcap-devel popt-devel e2fsprogs-devel mit-krb5-devel
- libldap-devel pam-devel acl-devel avahi-libs-devel tdb-devel talloc-devel cups-devel"
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --cross-compile --hostcc=cc
+	 --cross-execute=qemu-${XBPS_TARGET_QEMU_MACHINE}-static"
```

Comment:
Want to add `--cross-execute-qemu=...` as something that's added automatically to waf3 build style when the qemu build helper is used?

Reply:
Yes!

Reply:
`--hostcc=$CC_FOR_BUILD`